cli/tests/pcluster/validators/test_ebs.py

```diff
@@ -10,116 +10,194 @@
 # limitations under the License.
 import pytest
 
-from pcluster.schemas.cluster_schema import SharedStorageSchema
+from pcluster.models.param import Param
+from pcluster.validators.ebs_validators import (
+    EbsVolumeIopsValidator,
+    EbsVolumeSizeSnapshotValidator,
+    EbsVolumeThroughputIopsValidator,
+    EbsVolumeThroughputValidator,
+    EbsVolumeTypeSizeValidator,
+)
 from tests.pcluster.validators.utils import assert_failure_messages
 
 
 @pytest.mark.parametrize(
-    "section_dict, expected_message",
+    "volume_type, volume_throughput, expected_message",
     [
-        ({"VolumeType": "gp3", "Throughput": 125}, None),
-        (
-            {"VolumeType": "gp3", "Throughput": 100},
-            "Throughput must be between 125 MB/s and 1000 MB/s when provisioning gp3 volumes.",
-        ),
-        (
-            {"VolumeType": "gp3", "Throughput": 1001},
-            "Throughput must be between 125 MB/s and 1000 MB/s when provisioning gp3 volumes.",
-        ),
-        ({"VolumeType": "gp3", "Throughput": 125, "Iops": 3000}, None),
-        (
-            {"VolumeType": "gp3", "Throughput": 760, "Iops": 3000},
-            "Throughput to IOPS ratio of .* is too high",
-        ),
-        ({"VolumeType": "gp3", "Throughput": 760, "Iops": 10000}, None),
-        (
-            {"VolumeType": "gp3", "Throughput": 1001, "Iops": 2900},
-            [
-                "IOPS rate must be between 3000 and 16000 when provisioning gp3 volumes.",
-                "Throughput must be between 125 MB/s and 1000 MB/s when provisioning gp3 volumes.",
-            ],
```

Comment:
Are we going to test multiple failure messages? My concern was that now we are trying to report multiple failures at once. By testing validators individually,  can we make sure these wrong inputs do not trigger any exceptions during the execution of the chain of the validators? If not, are we going to test it somewhere else?

Reply:
For multiple failures from the same validator we should extend these unit tests.
For multiple failures not related to the same validator we can create more unit tests loading the section as before.
The test for the "chain of validators" should be validated by the unit tests related to the priority feature.